torchinductor/triton_ops/matmul.py

```diff
@@ -121,13 +122,14 @@ def grid(META):
             b.stride(1),
             c.stride(0),
             c.stride(1),
+            allow_tf32=allow_tf32,
             GROUP_M=8,
             ACC_TYPE=ACC_TYPE,
         )
 
     @staticmethod
-    def forward(a, b, out):
-        return _matmul_out._call(a, b, out)
+    def forward(a, b, out, allow_tf32=torch.backends.cuda.matmul.allow_tf32):
```

Comment:
This will read the value at `import` time, not at runtime.   Default values get evaluated when a function is defined.

Reply:
That makes sense. set default as True